NEWS.md

```diff
@@ -1061,7 +1061,11 @@ Deprecated or removed
   * The fallback method `^(x, p::Integer)` is deprecated. If your type relied on this definition,
     add a method such as `^(x::MyType, p::Integer) = Base.power_by_squaring(x, p)` ([#23332]).
 
-  * `wait` and `fetch` on `Task` now resemble the interface of `Future`
+  * `wait` and `fetch` on `Task` now resemble the interface of `Future`.
+
+  * `showcompact(io, x...)` has been deprecated in favor of
+    `show(IOContext(io, :compact => true), x...)` ([#26080]).
+    Use `sprint(showcompact, x...)` instead of `sprint(show, x..., context=:compact => true)`.
```

Comment:
That's the wrong way around, isn't it?

Reply:
Thanks. I had originally written "is deprecated in favor of", but I changed it because it's not really a separate deprecation, mostly an advice. Will fix in the next round.

Reply:
Fixed.